keyboards/launchpad/rev1/config.h

```diff
@@ -15,6 +15,8 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#pragma once
```

Comment:
If you're going to add this, please also remove the include guards.

Reply:
oops. fixed.